tests/perfalert/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+@pytest.fixture
+def job_from_try(test_job):
```

Comment:
If I understand how pytest work, a test that uses both the `test_job` and `job_from_try` fixtures, will get the same object for both, and it will be marked as being on try. I'm not sure if that is worth worrying about; but I expect it would be surprising.